config/locales/active_admin.de.yml

```diff
@@ -629,6 +631,8 @@ de:
       as: "Artikeltypen"
       general: "Allgemein"
       article_fields: "Artikelfelder"
+      default_template: "Standard Layout"
+      templates: "Verfügbare Templates"
```

Comment:
Wollen wir hier noch vereinheitlichen? Also Standardlayout und verfügbare Layouts?
Bei der Artikelauswahl wird ja auch von Layouts gesprochen:
<img width="304" alt="bildschirmfoto 2018-03-09 um 13 24 41" src="https://user-images.githubusercontent.com/90779/37207431-3f43c378-239d-11e8-9fd7-74cf619c6d0f.png">

Reply:
Stehe auf dem Schlauch... was genau würdest du hier anpassen wollen? Hier wählt man ja ein Layout aus in dem die Seite gerendert werden soll und in dem DropDown sind (natrürlich) nur noch die verfügbaren Layouts drin. Das StandardLayout taucht hier in der vorm ja nur dahingehen auf, das es in der DropDownListe vorausgewählt ist.

Reply:
oder meinst du die Benennung von Template <> Layout? Beim Artikeltyp wählt man ja jetzt neu das Template aus. Mittelfristig würde ich auch beim Artikel das Template auswählen wollen, noch ist das aber das Layout was hier gewählt wird.

Reply:
Ich meine, wo ich markiert habe

```
+      default_template: "Standard Layout"
+      templates: "Verfügbare Templates"
```

Das diese das gleiche meinen aber jeweils anders bezeichnet ist.
Der Screenshot sollte nur verdeutlichen, dass es im Artikel Layout heißt. Also solange wir nicht refactoren zu Template, auch in den Locales eher beides einheitlich Layout sein sollte, damit es passt und ein Nutzer nicht von Layout/Template verwirrt werden könnte.